src/main/java/com/google/sps/servlets/RecipeServlet.java

```diff
@@ -151,4 +162,18 @@ private String getDetailedRecipe(String recipeID) throws IOException {
 
     return "Exception";
   }
+
+  private void deleteComments(String recipeID) {
+    try {
+      ApiFuture<QuerySnapshot> future = DBReferences.comments(recipeID).get();
+      List<QueryDocumentSnapshot> documents = future.get().getDocuments();
```

Comment:
could inline 'documents' into next line, 
for (QDS document : queryFuture.get().getDocuments())
to simplify readability